src/validators/js-expression-validator.ts

```diff
@@ -23,7 +23,7 @@ export const ExpressionValidator: FormFieldValidator = {
         return evaluateExpression(
           config[key],
           { value: field, type: 'field' },
-          config.fields,
+          config.formFields,
```

Comment:
Was the issue the key/naming being used?

Reply:
Affirmative!